cognite/neat/core/rules/exporter/rules2graphql.py

```diff
@@ -156,9 +81,9 @@ def _define_fields(property_definitions: list[Property]) -> dict[str, GraphQLFie
         return gql_field_definitions
 
     gql_type_definitions: dict = {}
-    for class_, properties in transformation_rules.get_classes_with_properties().items():
-        gql_type_definitions[repair_name(class_, "class", fix_casing=fix_casing)] = GraphQLObjectType(
-            repair_name(class_, "class", fix_casing=fix_casing),
+    for class_, properties in get_classes_with_properties(transformation_rules).items():
+        gql_type_definitions[class_] = GraphQLObjectType(
+            class_,
             lambda properties=properties: _define_fields(properties),
```

Comment:
@doctrino I intend to drop `graphql-core` package in new branch, as I can achieve same with much less code using jinja2. So this whole module will be rewriten. I did not particularly like it since the begining.